__tests__/app.test.js

```diff
@@ -143,3 +129,65 @@ describe("GET /api/articles/", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("200: Responds with an array of comments from the specified article", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id");
+          expect(comment).toHaveProperty("votes");
+          expect(comment).toHaveProperty("created_at");
+          expect(comment).toHaveProperty("author");
+          expect(comment).toHaveProperty("body");
+          expect(comment).toHaveProperty("article_id");
+        });
+      });
+  });
+
+  test("200: Responds with an array of comments from the specified article sorted by created_at (DESC)", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+
+  test("404: Responds with an 'article or comments not found' error message (article doesn't exist)", () => {
+    return request(app)
+      .get("/api/articles/9999/comments")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("article or comments not found");
+      });
+  });
+
+  test("404: Responds with an 'article or comments not found' error message (article exists, no comments)", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("article or comments not found");
+      });
+  });
```

Comment:
Good idea with the above test - but in the case of an article existing then we wouldn't want to return an error. The article exists there are just no comments associated with it ...yet. Much like opening an article and seeing 'be the first to leave a comment!' 

In this case we'd actually want to just return an empty array and a 200 - as long as the article exists. 

You will need to find a way to check whether the article does exist. 

(this is actually the content of tomorrow's lecture, so if you plan to come back to it then, fine by me!)